clang/lib/CConv/ConstraintBuilder.cpp

```diff
@@ -347,11 +347,12 @@ class GlobalVisitor : public RecursiveASTVisitor<GlobalVisitor> {
       if (G->hasInit()) {
         CB.constrainLocalAssign(nullptr, G, G->getInit());
       }
-      // If the location of the previous RecordDecl and the current VarDecl are
-      // the same, this implies an inline struct as per Clang's AST, so set a
-      // flag in ProgramInfo to indicate that this variable should be
-      // constrained to wild later
-      if (lastRecordLocation == G->getBeginLoc().getRawEncoding()) {
+      // If the location of the previous RecordDecl and the current VarDecl
+      // coincide with one another, we constrain the VarDecl to be wild
+      // in order to allow the fields of the RecordDecl to be converted
+      int BeginLoc = G->getBeginLoc().getRawEncoding();
+      int EndLoc = G->getEndLoc().getRawEncoding();
```

Comment:
Make these unsigned. lastRecordLocation is unsigned.

Reply:
Sure 👍